sspl_test/alerts/node/test_node_disk_actuator.py

```diff
@@ -15,72 +15,76 @@
 
 # -*- coding: utf-8 -*-
 
-from common import check_sspl_ll_is_running, get_fru_response, send_node_controller_message_request
+from common import actuator_response_filter, get_node_controller_message_request
+from framework.base.testcase_base import TestCaseBase
 
 
-UUID="16476007-a739-4785-b5c6-f3de189cdf18"
-
-def init(args):
-    pass
-
-def test_node_disk_module_actuator(agrs):
-    check_sspl_ll_is_running()
+class NodeDiskActuatorTest(TestCaseBase):
+    UUID = "16476007-a739-4785-b5c6-f3de189cdf18"
     instance_id = "*"
     resource_type = "node:fru:disk"
```

Comment:
Please rebase the branch with latest main. Resource type has been changed for many sensors/actuator